EIPS/eip-5069.md

```diff
@@ -1,44 +1,118 @@
+ - **Upholding Quality**: Upholding a measure of minimally-subjective quality for each proposal as defined by its target audience.
 
-- Good communication skills
-- Ability to handle contentious discourse
-- 1-5 spare hours per week
+    By adhering to defined criteria, we promote the development of high-quality and relevant proposals that drive the evolution of Ethereum.
 
-The best available resource to understand the EIP process is [EIP-1](./eip-1.md). Anyone desirous of becoming an EIP editor MUST understand this document. Afterwards, participating in the EIP process by commenting on and suggesting improvements to PRs and issues will familliarize the procedure, and is recommended. The contributions of newer editors should be monitored by other EIP editors.
+### What we Don't
 
-Anyone meeting the above requirements may make a pull request adding themselves as an EIP editor and adding themselves to the editor list at `config/eip-editors.yml` and in [EIP-1](./eip-1.md). If every existing EIP editor approves, the author becomes a full EIP editor. This should notify the editor of relevant new proposals submitted in the EIPs repository, and they should review and merge those pull requests.
+On the other hand, we do _not_:
 
-### Special Merging Rules for this EIP
+ - **Decide Winners**: If there are multiple competing proposals, we will publish all of them. We are not in the business of deciding what is the right path for Ethereum, nor do we believe that there is One True Way to satisfy a need.
 
-This EIP MUST have the same rules regarding changes as [EIP-1](./eip-1.md).
+ - **Assert Correctness**: While we might offer technical feedback from time to time, we are not experts nor do we vet every proposal in depth. Publishing a proposal is not an endorsement or a statement of technical soundness.
 
-## Rationale
+ - **Manage**: We do not track implementation status, schedule work, or set fork dates or contents.
 
-- "6 months" was chosen as the cutoff for denoting `Stagnant` EIPs terminally-`Stagnant` arbitrarily.
-- This EIP requires special merging rules for the same reason [EIP-1](./eip-1.md) does.
+ - **Track Registries**: We want all proposals to eventually become immutable, but a registry will never get there if anyone can keep adding items. To be clear, exhaustive and/or static lists are fine. 
+
+ - **Provide Legal Advice**: Trademarks, copyrights, patents, prior art, and other legal matters are the responsibility of authors and implementers, not EIP Editors. We are not lawyers, and while we may occasionally make comments touching on these areas, we cannot guarantee any measure of correctness.
+
+Documenting all of the things we would not do is impossible, and the above are just a few examples. We reserve the right to do less work whenever possible!
+
+## Structure
+
+### EIP Editors
+
+We, the Editors, consist of some number of EIP Editors and one Janitor elected by and from the EIP Editors.
```

Comment:
I really do like the term "Janitor" because other terms, like "moderator", "chairperson", "editor-in-chief" all imply some amount of authority.

I like the implication that the editor in this role "takes out the trash" and does other somewhat messy work. Being the Janitor isn't an honour, but an undesirable position to be in.

Reply:
I went wit "Keeper of Consensus" (or Keeper for short.)